tests/validators/invalid_urls.txt

```diff
@@ -50,3 +50,4 @@ http://[::1:2::3]:8080/
 http://[]
 http://[]:8080
 http://example..com/
+://example.com
```

Comment:
This seems to already fail on master and 1.8.x without your change so there doesn't seem to be a regression unless you can provide a different input.